.travis.yml

```diff
@@ -63,18 +49,19 @@ jobs:
     # GitHub Pages Deployment: https://docs.travis-ci.com/user/deployment/pages/
     - stage: Docs
       python: "3.7"
-      if: type != cron
+
+#      if: type != cron and $TRAVIS_PULL_REQUEST = True
```

Comment:
This should be uncommented, right ? And the condition probably wont work neither, please see the docs : https://docs.travis-ci.com/user/conditions-v1#integration